frontend/src/store/wallet.ts

```diff
@@ -362,22 +369,41 @@ export default function useWalletStore() {
   // ------------------------------------- Computed parameters -------------------------------------
   // "True" computed properties, i.e. derived from this module's state
 
-  const currentChain = computed(() => {
-    return getChainById(network.value?.chainId || 1);
-  });
+  const chainId = computed(() => network.value?.chainId); // returns a number
+  const currentChain = computed(() => getChainById(chainId.value || 1));
 
   const isSupportedNetwork = computed(() => {
-    if (!network.value) return true; // assume valid if we have no network information
-    return supportedChainIds.includes(network.value.chainId);
+    if (!network.value || !chainId.value) return true; // assume valid if we have no network information
+    return supportedChainIds.includes(chainId.value);
   });
 
+  const getNativeTokenMinSend = (chainId: number): string => {
+    switch (chainId) {
+      case 137:
+        return parseUnits('0.15', 'ether').toString(); // Polygon
+      default:
+        return parseUnits('0.02', 'ether').toString(); // everything else
```

Comment:
How do these compare to the prior defaults / how did we come up with these?

Reply:
These are the current values returned by the backend, calculated using the formula:

Gas Price x Estimated Gas Limit x Scale Factor

I'll add a note about it